components/registry-facade-api/go/config/config.go

```diff
@@ -80,10 +80,10 @@ type RedisCacheConfig struct {
 
 	SingleHostAddress string `json:"singleHostAddr,omitempty"`
 
-	MasterName    string   `json:"masterName,omitempty"`
-	SentinelAddrs []string `json:"sentinelAddrs,omitempty"`
```

Comment:
Since it's the API change, would it be good to mark this field as deprecated?

Reply:
Why? This is an experimental setting only used by us in SaaS.